functors_task_1/student/include/impl.h

```diff
@@ -16,16 +26,24 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
 {
+    if (beer == BeerBrand::Corona || beer == BeerBrand::HoeGaarden) {
```

Comment:
why do you need these branches? we can evaluate it directly in return

Reply:
You mean if (beer == BeerBrand::Corona) return true or to make a short if?

Reply:
right, just return beer == ...